count_the_number_of_days_of_specific_month_18.py

```diff
@@ -0,0 +1,20 @@
+import numpy as np
+
+
+def count_the_number_of_days_of_specific_month(date1, date2):
```

Comment:
Use type hints and return types -docum
https://www.geeksforgeeks.org/duck-typing-in-python/
https://stackoverflow.com/questions/125367/dynamic-type-languages-versus-static-type-languages
https://docs.python.org/3/library/typing.html
https://realpython.com/lessons/pros-and-cons-type-hints/